lambda_functions/deploy.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    echo "Usage: ./deploy.sh <lambda_function_name>"
+    exit 1
+fi
+
+FUNCTION_NAME="$1"
+
+cd "$FUNCTION_NAME"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)